Thread border router: add intro (#42620)
home-assistant/home-assistant.github.io: source/_integrations/thread.markdown

### About Thread border routers

> A Thread border router is a network device that connects a Thread mesh network (consisting of low-power IoT devices) to other IP networks such as Ethernet or Wi-Fi.

Reviewer: "consisting of low-power IoT devices" is not wrong, but I'd then mention they have to be Thread devices.
I'm not sure we even have to be that verbose though. Maybe something more simple works as well? E.g.:

```suggestion
A Thread border router is a device that connects a Thread mesh network to your local network via Wi-Fi or Ethernet.
```
or
```suggestion
A Thread border router is a device that connects a Thread mesh network to your home's IP network via Wi-Fi or Ethernet.
```

It uses a radio chip to communicate with low-power Thread devices in the mesh, and routes their traffic to and from your local network.
The Thread border router does not control devices. Device control is handled by application layer protocols like Matter or Apple HomeKit.
